src/chainparamsbase.h

```diff
@@ -9,6 +9,8 @@
 #include <string>
 #include <vector>
 
+#define CHAINPARAMS_CONF_FILENAME "chainparams.conf"
```

Comment:
Oops, this are rests from when the PR only allowed to set the custom chainparams parameter from its own specific conf file, but not from the command line options nor the regular config file (to separate concerns). If that's preferred, it shouldn't be hard to do again on top of #9494